apps/site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -425,15 +422,15 @@ To publish a block on the Hub take the following steps.
 ### Publish from your terminal
 
 - Run `npm run build` or `yarn build` to create a production build of your block (it will appear in the `dist` folder)
-- run `npx blockprotocol publish` to generate a `.blockprotocolrc` file when prompted
+- run `npx blockprotocol@latest publish` to generate a `.blockprotocolrc` file when prompted
 - Replace the placeholder key in that file with your API key
-- now run `npx blockprotocol publish` again
+- now run `npx blockprotocol@latest publish` again
 - See your block on the Hub!
 
 ### Updating your block
 
 You can update your published block at any time by running
-`npm run build && npx blockprotocol publish` or `yarn build && npx blockprotocol publish`
+`npm run build && npx blockprotocol@latest publish` or `yarn build && npx blockprotocol@latest publish`
```

Comment:
Can we swap this around, we mention `yarn` first elsewhere, and I think it's the default setup in the templates

Reply:
[Make ordering of npm run and yarn consistent](https://github.com/blockprotocol/blockprotocol/pull/1086/commits/a1cff771d71049eb25ec12f5479bf8874674a2af)